nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2876,40 +2980,40 @@ FlowFile Repository, if also on that disk, could become corrupt. To avoid this s
  +
 *NOTE*: Multiple content repositories can be specified by using the *_nifi.content.repository.directory._* prefix with unique suffixes and separate paths as values. +
  +
 For example, to provide two additional locations to act as part of the content repository, a user could also specify additional properties with keys of: +
  +
-nifi.content.repository.directory.content1=/repos/content1 +
-nifi.content.repository.directory.content2=/repos/content2 +
+`nifi.content.repository.directory.content1=/repos/content1` +
+`nifi.content.repository.directory.content2=/repos/content2` +
  +
 Providing three total locations, including  `nifi.content.repository.directory.default`.
-|nifi.content.repository.archive.max.retention.period|If archiving is enabled (see nifi.content.repository.archive.enabled below), then
+|`nifi.content.repository.archive.max.retention.period`|If archiving is enabled (see `nifi.content.repository.archive.enabled` below), then
 this property specifies the maximum amount of time to keep the archived data. The default value is `12 hours`.
-|nifi.content.repository.archive.max.usage.percentage|If archiving is enabled (see nifi.content.repository.archive.enabled below), then this property must have a value that indicates the content repository disk usage percentage at which archived data begins to be removed. If the archive is empty and content repository disk usage is above this percentage, then archiving is temporarily disabled. Archiving will resume when disk usage is below this percentage. The default value is `50%`.
-|nifi.content.repository.archive.enabled|To enable content archiving, set this to _true_ and specify a value for the `nifi.content.repository.archive.max.usage.percentage` property above. Content archiving enables the provenance UI to view or replay content that is no longer in a dataflow queue. By default, archiving is enabled.
-|nifi.content.repository.always.sync|If set to `true`, any change to the repository will be synchronized to the disk, meaning that NiFi will ask the operating system not to cache the information. This is very expensive and can significantly reduce NiFi performance. However, if it is `false`, there could be the potential for data loss if either there is a sudden power loss or the operating system crashes. The default value is `false`.
-|nifi.content.viewer.url|The URL for a web-based content viewer if one is available. It is blank by default.
+`|`nifi.content.repository.archive.max.usage.percentage`|If archiving is enabled (see `nifi.content.repository.archive.enabled` below), then this property must have a value that indicates the content repository disk usage percentage at which archived data begins to be removed. If the archive is empty and content repository disk usage is above this percentage, then archiving is temporarily disabled. Archiving will resume when disk usage is below this percentage. The default value is `50%`.
```

Comment:
I think there is an accidental initial backtick before the column delimiter.

Reply:
Good catch.